test/kdump_client_test.rb

```diff
@@ -0,0 +1,78 @@
+require_relative "./test_helper"
+
+# Let's "dirty-mock" Yast::KdumpComplexInclude to avoid not desired dialogs during test
```

Comment:
Please, this hack deserves a FIXME note :-)

Reply:
Sure, although I don't know what say there :P